ouroboros-consensus/src/Ouroboros/Consensus/Node/Run/Byron.hs

```diff
@@ -28,16 +33,22 @@ instance ByronGiven => RunNode (ByronBlockOrEBB ByronConfig) where
   nodeIsEBB              = \blk -> case unByronBlockOrEBB blk of
     Cardano.Block.ABOBBlock _    -> False
     Cardano.Block.ABOBBoundary _ -> True
-  nodeEpochSize          = \_ _ -> return 21600 -- TODO #226
+
+  -- The epoch size is fixed and can be derived from @k@ by the ledger
+  -- ('kEpochSlots').
+  nodeEpochSize          = const
```

Comment:
How about `\_proxy _epochNo -> return ...`

Reply:
I'll sacrifice the nice alignment for some readability :slightly_smiling_face: